Makefile

```diff
@@ -51,24 +51,15 @@ PACKAGES=$(shell go list ./...)
 PACKAGES_WITH_TESTS = $(shell go list -f '{{if len .XTestGoFiles}}{{.ImportPath}}{{end}}' ./... \
 							&& go list -f '{{if len .TestGoFiles}}{{.ImportPath}}{{end}}' ./...)
 
-checkstyle: ## Run quick checkstyle (govet + goimports (fail on errors))
-	@echo "==> Running govet"
-	go vet $(PACKAGES) || exit 1
-	@echo "==> Running goimports"
-	goimports -l -w .
-	@echo "==> SUCCESS";
```

Comment:
Why remove this?

Reply:
goimports is included in linter we use. so this target is just a dup in functionality.